src/trezorClient.py

```diff
@@ -201,7 +201,6 @@ def prepare_transfer_tx_bulk(self, caller, rewardsArray, dest_address, tx_fee, i
         c_name = "PIVX"
         if isTestnet:
             c_name += " Testnet"
-        coin = coins.by_name[c_name]
```

Comment:
Nice catch. This is done again in SignTxSign.
We can remove the `c_name` variable here as well.